content/en/platform/corda/4.6/open-source/node-explorer.md

```diff
@@ -143,7 +143,7 @@ To see a geographical view of your network, click **Network** in the left-hand s
 
 {{< note >}}
 
-The geographical location of each node is based on the [`locality` property of the node's registered CordaX500 name](https://api.corda.net/api/corda-os/4.6/html/api/kotlin/corda/net.corda.core.identity/-corda-x500-name/index.html)
+The geographical location of each node is based on the <a href="https://docs.r3.com/en/api-ref/corda/4.6/open-source/kotlin/corda/net.corda.core.identity/-corda-x500-name/index.html">`locality` property of the node's registered CordaX500 name</a>.
 
 {{< /note >}}
 
```

Comment:
One of those weird formatting issues on this page: 
![image](https://user-images.githubusercontent.com/69467840/137884227-24fecab8-1572-446f-93e6-2ba85ae317b4.png)